redaxo/src/core/lib/util/i18n.php

```diff
@@ -7,8 +7,17 @@
  */
 class rex_i18n
 {
+    /**
+     * @var string[]
+     */
     private static $locales = [];
+    /**
+     * @var string[]
+     */
     private static $directories = [];
+    /**
+     * @var boolean[string] Holds which locales are loaded. keyed by locale
```

Comment:
Ist das so wirklich eine gängige Notation?

Reply:
das ist, was ich so gefunden hab: https://stackoverflow.com/questions/2713710/comment-associative-array-in-php-documentor

Reply:
D.h. Kann ich das so übernehmen?